addons/website_slides_survey/static/src/xml/website_slides_fullscreen.xml

```diff
@@ -2,11 +2,16 @@
 <templates>
 
     <t t-name="website.slides.fullscreen.certification">
-        <div class="justify-content-center align-self-center">
+        <div class="justify-content-center align-self-center" style="text-align:center;">
+            <div t-if="widget.get('slide').category === 'certification' &amp;&amp; widget.get('slide').survey_url">
```

Comment:
As explained in an earlier comment, the goal is to always display that message if the user has the sufficient access rights.

The survey_url should probably be added within the data alongside survey specific information.
(See website_slides_survey/views/website_slides_templates_lesson_fullscreen.xml).